version/version.go

```diff
@@ -11,11 +11,14 @@ var BuildDate string
 // GitCommit is the commit hash when the binary was built
 var GitCommit string
 
-// Version is the version of the binary
-var Version string
+// MICVersion is the version of the MIC component
+var MICVersion string
+
+// NMIVersion is the version of the NMI component
+var NMIVersion string
 
 // PrintVersionAndExit prints the version and exits
 func PrintVersionAndExit() {
```

Comment:
Can this be used in case of -v command line option for both mmi and mic. Would be a good way to find out the comparable nmi/mic.

Reply:
do we really want to exit the process after printing the version?

Reply:
I was thinking if so if some one runs 

nmi -v
or 
mic -v

this is the only which needs to be called. In that case the exit can be left there as is - thoughts ?

Also wondering why the fmt was used here originally, does this assume that blog is not initialized or some other rule to be followed in version.go ?